src/index.js

```diff
@@ -0,0 +1,277 @@
+const tangramReference = require('tangram-reference').load();
+const color = require('./color.js');
+
+//getReferenceIndexedByCSS returns a reference based on ref indexed by the CSS property name instead of the reference original property name
+//it does a shallow copy of each property, therefore, it's possible to change the returned reference by changing the original one, don't do it
+function getReferenceIndexedByCSS(ref) {
+    var newRef = {};
+    for (var symb in ref.symbolizers) {
+        for (var property in ref.symbolizers[symb]) {
+            newRef[ref.symbolizers[symb][property].css] = ref.symbolizers[symb][property];
+        }
+    }
+    return newRef;
+}
+const referenceCSS = getReferenceIndexedByCSS(tangramReference);
+
+
+function wrapFn(functionBody) {
+    return 'function (){var data=feature; var ctx={zoom: $zoom};' + functionBody + '}';
```

Comment:
I would use template literal as we have done in the rest of the code.

Reply:
Yes